### DISCUSSION TITLE
Upgrade process taking forever

### DISCUSSION THREAD

**Discussion by cos-gl** [nilsteampassnet/TeamPass#3889] [Q&A]
Hi!
First of all, thanks for this project! I was using TeamPass few years ago and now wanted to give a try again.
I don't know if this could be an issue or not, since I'm not experienced in usign recent versions of TeamPass.
I just installed from scratch the latest version (it is a test installation) usign docker-compose (following instructions on the repo readme).
The installation process was fine. Was able to login using admin account and create a user.
When trying to assign a folder to the user a process is taking forever... Reading some other recent issue the solution proposed was to update to the latest commit so I tried updating tempass-html folder using 
```
git clone https://github.com/nilsteampassnet/TeamPass.git
```
I set the permissions for the right user, in the folder, and went to the web interface
```
http://teampass.mydomain.local:8080/install/upgrade.php
```
After starting the process a rotating cog icon appears and doesn't stop...

Looking at borwser console (dev tools), this is the error

```
Uncaught SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data
    complete http://teampass.lucigroup.loc:8080/install/upgrade.php:216
    jQuery 6
    <anonymous> http://teampass.lucigroup.loc:8080/install/upgrade.php:208
    jQuery 2
upgrade.php:216:26
![TeamPass2023-09-28 101320](https://github.com/nilsteampassnet/TeamPass/assets/146211759/6c73e979-d81d-4246-ac92-51175c1616b3)

```

Am I doing something wrong?

Thanks!

### Server configuration
**Operating system**: Ubuntu 22.04

**Web server:** ngnix

**Database:** mariadb

**PHP version:** 8.2 

**Teampass version:** 3.0.10

**HTTP/HTTPS**: I have modified the docker-compose to expose port 80 (port 8080 outside of container) of teampass-html container, just not to mess with certificates since is a test installation

### Client configuration
**Browser:** Chrome 117 or Firefox 117

**Operating system:** Windows 11

![TeamPass2023-09-28 101320](https://github.com/nilsteampassnet/TeamPass/assets/146211759/f6305ef5-3ada-4a67-b948-b6944f2ddca1)

**Comment by nilsteampassnet** [comment]
@cos-gl 
Can I close this topic?
If I refer to #3870, it seems you managed it.

**Reply by cos-gl** [reply]
@nilsteampassnet not really... in my last attempt the issue was there. 
I haven's check latest release, I will do but in the meanwhile you can close the topic if you feel it's resolved.
